SourceX/main.cpp

```diff
@@ -19,13 +19,49 @@ static std::string build_cmdline(int argc, char **argv)
 		}
 		str += argv[i];
 	}
+
 	return str;
 }
 
 int main(int argc, char **argv)
 {
 	auto cmdline = build_cmdline(argc, argv);
-	return dvl::WinMain(NULL, NULL, (char *)cmdline.c_str(), 0);
-}
-
 
+	// TODO (cmhobbs)
+	// - [X] catch -h and --help in cmdline()
+	// - [X] add debug output options
+	//   - ref: https://github.com/diasurgical/devilutionX/blob/master/docs/debug.md
+	// - [ ] add options from GHI
+	//   - ref: https://github.com/diasurgical/devilutionX/issues/219
+	//   - [ ] -h and --help to print help options
+	//   - [ ] -v and --version to show version or git branch/rev/commit
+	//   - [ ] --data-dir path to diabdat.mpq
+	if ((cmdline == "-h") || (cmdline == "--help")) {

```

Comment:
Maybe it makes more sense to stuff all this into its own function?

Reply:
please choose one, sometimes you write "enable", sometimes "enables" lol :P

Reply:
I took the descriptions directly from docs/debug.md.  Which do we prefer?

Reply:
I'd go with "enable" because it's shorter 🤣

Reply:
Great!  I'll make the changes.  Thanks for the input!